aleph/tests/test_entities_api.py

```diff
@@ -351,3 +350,204 @@ def test_entity_tags(self):
         assert len(results) == 1, results
         assert results[0]['value'] == '+491769817271', results
         validate(res.json['results'][0], 'EntityTag')
+
+    def test_undelete(self):
```

Comment:
Looking at it now: is undelete the same as update? Should we calm down and make update do an undelete by default?

Reply:
if we only `undeleted` the entity then I agree that it could be done inside `update` endpoint. But we create new entities too. So undelete is more of an `upsert`. It seems very random to do that in `update` if we're doing CRUD according to REST in all other places.